pom.xml

```diff
@@ -363,11 +357,30 @@
             <id>spark-3.2</id>
             <properties>
                 <spark.version>${spark-32.version}</spark.version>
-                <avro.version>1.10.2</avro.version>
                 <confluent.version>6.2.1</confluent.version>
             </properties>
         </profile>
-
+        <profile>
+            <id>spark-3.3</id>
+            <properties>
+                <spark.version>${spark-33.version}</spark.version>
+                <confluent.version>6.2.1</confluent.version>
+            </properties>
+        </profile>
+        <profile>
+            <id>spark-3.4</id>
+            <properties>
+                <spark.version>${spark-34.version}</spark.version>
+                <confluent.version>6.2.1</confluent.version>
+            </properties>
+        </profile>
+        <profile>
+            <id>spark-3.5</id>
+            <properties>
+                <spark.version>${spark-35.version}</spark.version>
+                <confluent.version>6.2.1</confluent.version>
+            </properties>
+        </profile>
```

Comment:
`confluent.version` seems to be the same everywhere.

Reply:
Yes, I guess we can set it as a constant then. If we want to make it dynamic it in the future, we can always change it